src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/DocumentTest.java

```diff
@@ -33,13 +34,27 @@ public void fromScannableItem_maps_to_document_correctly() {
 
     @Test
     public void fromScannableItem_maps_document_type_correctly() {
-        ScannableItem cherishedScannableItem = scannableItem(DOCUMENT_TYPE_CHERISHED);
-        ScannableItem otherScannableItem = scannableItem(DOCUMENT_TYPE_OTHER);
-        ScannableItem sscs1ScannableItem = scannableItem("SSCS1");
+        Map<String, String> expectedTypes = new HashMap<>();
+        expectedTypes.put("Cherished", CCD_DOCUMENT_TYPE_CHERISHED);
+        expectedTypes.put("cherished", CCD_DOCUMENT_TYPE_CHERISHED);
+        expectedTypes.put("CHERISHED", CCD_DOCUMENT_TYPE_CHERISHED);
+        expectedTypes.put("Other", CCD_DOCUMENT_TYPE_OTHER);
+        expectedTypes.put("other", CCD_DOCUMENT_TYPE_OTHER);
+        expectedTypes.put("OTHER", CCD_DOCUMENT_TYPE_OTHER);
+        expectedTypes.put("SSCS1", CCD_DOCUMENT_TYPE_OTHER);
+        expectedTypes.put("sscs1", CCD_DOCUMENT_TYPE_OTHER);
+        expectedTypes.put("Sscs1", CCD_DOCUMENT_TYPE_OTHER);
```

Comment:
did you consider Guava's
```
ImmutableMap.of(
  k1, v1,
  k2, v2,
  k3, v3
)
```
to avoid all the `expectedTypes.put(`?

Reply:
My first choice. Max 5 elements.

Reply:
how about? https://google.github.io/guava/releases/19.0/api/docs/com/google/common/collect/ImmutableMap.Builder.html

Reply:
That one would do. Still, this test will be gone by the time I finish the story.